src/schema/v2/homeView/index.ts

```diff
@@ -16,11 +16,12 @@ const SectionsConnectionType = connectionWithCursorInfo({
 const SectionConnection: GraphQLFieldConfig<any, ResolverContext> = {
   type: SectionsConnectionType,
   args: pageable({}),
-  resolve: async (_parent, args, _context, _info) => {
+  resolve: async (_parent, args, context, _info) => {
     const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
 
-    const totalCount = STUB_SECTIONS.length
-    const data = STUB_SECTIONS.slice(offset, offset + size)
+    const sections = await getSectionsForUser(context)
```

Comment:
Slight evolution of this placeholder section data: from a stubbed array identical for everyone → to a slightly varying array based on the user's attributes.